symfony/flex/1.0/git/config

```diff
@@ -0,0 +1,5 @@
+[core]
+	repositoryformatversion = 0
+	filemode = true
```

Comment:
this has a drawback: on windows, the default here should be `false` (otherwise, git would change files to executable when committing on windows due to Windows not supporting Unix filemode)

Reply:
should we just remove the line?